lib/EnsEMBL/REST/Controller/LD.pm

```diff
@@ -43,7 +43,7 @@ sub id_GET {
   try {
     $LDFeatureContainer = $c->model('LDFeatureContainer')->fetch_LDFeatureContainer_variation_name($id, $population_name);
   } catch {
-    $c->go('ReturnError', 'from_ensembl', []) if $_ =~ /STACK/;    
+    $c->go('ReturnError', 'from_ensembl', [qq{$_}]) if $_ =~ /STACK/;    
```

Comment:
I would suggest adding whitespace around the qq{} block, too many brackets in a row :)